updating details on EOL (#1889)
aiven/devportal: docs/platform/reference/eol-for-major-versions.rst

-  Email reminders will be sent to customers on a monthly cadence. On
   the month of the EOL date, the cadence shifts to weekly reminders.

What happens to the outdated services beyond EOL?
-------------------------------------------------

Continued use of outdated services means that they no longer offer our
customers the level of protection their business needs. Therefore, by
following the upstream project's EOL schedule, we ensure that Aiven
services are always running on supported versions of the open source

> software. The following two scenarios will be observed beyond EOL:

Reviewer: ```suggestion
software. After the EOL date:
```

> 1. If the service is powered-on, it will be auto-upgraded to the latest version.

Reviewer: ```suggestion
1. If the service is powered on, it's auto-upgraded to the latest version.
```

> 2. If the service is powered-off, it will be deleted.

Reviewer: ```suggestion
2. If the service is powered off, it's deleted.
```

Author: Thanks for the suggestions. It has been edited as such.
